packages/web/components/chart/home-chart.tsx

```diff
@@ -0,0 +1,89 @@
+import { AreaData, MouseEventParams, Time } from "lightweight-charts";
```

Comment:
Why not visx? We already import it.

Reply:
We had started out using visx, but there are behaviors that it doesn't handle natively, and in the reference indicated they were using lightweight-chart.

I can think whether it is possible to do the same things with visx, but right now the biggest problem is that the handling of asset x is a bit buggy on visx